ddl/ddl_api.go

```diff
@@ -7564,6 +7584,120 @@ func checkIgnorePlacementDDL(ctx sessionctx.Context) bool {
 	return false
 }
 
+// CreateResourceGroup implements the DDL interface, creates a resource group.
+func (d *ddl) CreateResourceGroup(ctx sessionctx.Context, stmt *ast.CreateResourceGroupStmt) (err error) {
+	groupInfo := &model.ResourceGroupInfo{ResourceGroupSettings: &model.ResourceGroupSettings{}}
+	groupName := stmt.ResourceGroupName
+	groupInfo.Name = groupName
```

Comment:
Think about restrict the length of group name ?

Reply:
added